keras/src/activations/__init__.py

```diff
@@ -116,7 +116,7 @@ def get(identifier):
     if identifier is None:
         return linear
     if isinstance(identifier, dict):
-        obj = deserialize(identifier)
+        obj = serialization_lib.deserialize_keras_object(identifier)
```

Comment:
I think you should fix `deserialize` instead. External users using `deserialize` should be able to use activation layers.

But then `deserialize` is just an alias for `deserialize_keras_object`, so that's not very useful.

Reply:
Maybe we can just remove `module_objects=ALL_OBJECTS_DICT` argument when making this call from within the `deserialize` method?
I'm concerned that might cause some unintended behavior change for clients depending on that API.

Maybe we can swap out the `deserialize(identifier)` call for the `serialization_lib.deserialize_keras_object(identifier)` call here, and then later investigate/refactor the `deserialize` method separately.

Reply:
> Maybe we can just remove module_objects=ALL_OBJECTS_DICT argument when making this call from within the deserialize method?

Yes, exactly. That's what I meant by "fixing deserialize".

> I'm concerned that might cause some unintended behavior change for clients depending on that API.

All that `module_objects=ALL_OBJECTS_DICT` does is restrict the possibilities to the list of predefined activation **functions**. So it's not going to break existing use cases. But it will make some use cases that didn't work before work. I don't think anybody relies on the error happening for non-supported use case.

The only concern if we fix `deserialize` is that there no guarantee that the returned object is actually an activation function or layer. It could be anything. That's a bit weird.